creator-node/src/middlewares/premiumContent/premiumContentMiddleware.ts

```diff
@@ -2,15 +2,13 @@ import {
   sendResponse,
   errorResponseServerError,
   errorResponseForbidden,
+  errorResponseUnauthorized,
   errorResponseBadRequest
 } from '../../apiHelpers'
-import { recoverWallet } from '../../apiSigning'
 import { NextFunction, Request, Response } from 'express'
-import { isPremiumContentMatch } from '../../premiumContent/helpers'
-import { PremiumContentType } from '../../premiumContent/types'
-import { getRegisteredDiscoveryNodes } from '../../utils/getRegisteredDiscoveryNodes'
+import { PremiumContentAccessError } from '../../premiumContent/types'
 import type Logger from 'bunyan'
-import { Redis } from 'ioredis'
+import { PremiumContentAccessChecker } from '../../premiumContent/premiumContentAccessChecker'
 
 /**
```

Comment:
nit - this jsdoc is great but might be easier to follow as a less verbose, numbered/bulleted list of discrete steps/components